Tinyxml2Serializer.h

```diff
@@ -88,8 +88,8 @@ namespace qualisys_cpp_sdk {
     private:
         std::uint32_t mnMajorVersion;
         std::uint32_t mnMinorVersion;
-        void AddXMLElementBool(tinyxml2::XMLDocument* oXML, const char* tTag, const bool* pbValue, const char* tTrue = "True", const char* tFalse = "False");
```

Comment:
Suggestion: These arguments are used in other places. Consider keeping them
![image](https://github.com/user-attachments/assets/9e8a6a3f-ed0d-48a4-a210-aed2da23dfa5)

Reply:
Reintroduced the default-specified **tTrue == "True"** and **tFalse == "False"** to the function signatures. Updated the functions accordingly, and the tests still pass.